PSPrintSDK/OLFrameSelectionViewController.m

```diff
@@ -36,26 +36,26 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
     vc.assets = self.assets;
     if ([segue.identifier isEqualToString:@"Selected2x2FrameStyleSegue"]) {
         for (OLProduct *product in products){
-            if (product.templateType == kOLTemplateTypeFrame2x2){
+            if (product.productTemplate.quantityPerSheet == 4){
```

Comment:
Yup that should do it in this particular case. I don't anticipate same quantity for more than one frame template